descartes_trajectory/include/descartes_trajectory/joint_trajectory_pt.h

```diff
@@ -31,34 +31,24 @@
 namespace descartes_trajectory
 {
 
-//TODO add warning if non-zero tolerances are specified because initial implementation will only allow fixed joints
-struct JointTolerance
-{
-  JointTolerance(): lower(0.), upper(0.) {};
-  JointTolerance(double low, double high):
-    lower(std::abs(low)), upper(std::abs(high)) {};
-
-  double lower, upper;
-};
-
 struct TolerancedJointValue
 {
-  TolerancedJointValue() {};
-  TolerancedJointValue(double _nominal, double _tol_above, double _tol_below):
-    nominal(_nominal), tolerance(_tol_above, _tol_below) {};
-  TolerancedJointValue(double _nominal)
-  {
-    *this = TolerancedJointValue(_nominal, 0., 0.);
-  }
+  TolerancedJointValue(double nominal, double lower, double upper)
+    : nominal(nominal), lower(lower), upper(upper)
+  {}
+
+  TolerancedJointValue(double nominal)
+    : nominal(nominal), lower(nominal), upper(nominal)
+  {}
 
   double upperBound() const
   {
-    return nominal+tolerance.upper;
+    return upper;
   }
 
   double lowerBound() const
   {
-    return nominal-tolerance.lower;
+    return lower;
   }
 
   double range() const
```

Comment:
It looks like this method is no longer being used since now the range is computed from the bound arrays in the JointTrajectoryPoint